Model/ToolPathTxtWriter.cpp

```diff
@@ -0,0 +1,21 @@
+#include "stdafx.h"
+#include "ToolPathTxtWriter.h"
+
+ToolPathTxtWriter::ToolPathTxtWriter() {}
+ToolPathTxtWriter::~ToolPathTxtWriter() {}
+
+void ToolPathTxtWriter::generateFile(const QVector<GLfloat>& pathData)
+{
+	string fileData = "";
+	for (int i = 0; i < pathData.size()/3; i++)
+	{
+		string tempStr = to_string(pathData.at(i)) + " " + to_string(pathData.at(i + 1)) + " " + to_string(pathData.at(i + 2)) + "\n";
+		fileData += tempStr;
+	}
+	ofstream outputFile;
+	outputFile.open("D:/GeneratedToolPath.txt");
```

Comment:
I think it should be dynamic

Reply:
I thought to give the user quickest access to the path file generated by making it in here, which is also wrong. It's the user who decides where to download the file, because he/she is the one who has to store and use it.